### PR TITLE
Display notes and papers

### PR DESCRIPTION
**Pull request by zatteo**
Display notes and papers, in other words for notes and papers in the cipher `View`:
- add paper thumbnail (exemple in the screenshot is a screenshot itself)
- display paper metadata as `Fields`
- modify "Edit" and "Open in Cozy" links to redirect to mespapiers or notes app
- hide share action
- hide clone action
- add print action
- add download action
- UI and wording updates

image

### CODE REVIEW

File: libs/cozy/paperCipher.ts [hunk 1]
```diff
@@ -33,6 +33,11 @@ export const buildFilesQueryWithQualificationLabel = () => {
     "metadata.contractType",
     "metadata.refTaxIncome",
     "metadata.title",
+    "metadata.AObtentionDate",
+    "metadata.BObtentionDate",
+    "metadata.CObtentionDate",
+    "metadata.DObtentionDate",
+    "metadata.page",
```
Comment (zatteo) [inline libs/cozy/paperCipher.ts:40]: The query will be modified later it is still not decided if we include every metadata one by one or the full metadata object.

File: libs/cozy/fields.helper.ts [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+      formattedName = getTranslatedNameForOtherMetadata(label.name, { lang });
+      formattedValue = formatOtherMetadataValue(label.value, {
+        lang,
+        name: label.name,
+      });
+    } else if (metadataQualificationType === "contact" && paper.contacts.data.length > 0) {
+      formattedName = getTranslatedNameForContact({ lang });
+      formattedValue = formatContactValue(paper.contacts.data);
+    } else {
+      return;
```
Comment (Ldoppea) [inline libs/cozy/fields.helper.ts:67]: Why are you using a `forEach` with early return? Are there some labels without corresponding qualificationtype?

File: apps/browser/src/popup/scss/misc.scss [hunk 3]
```diff
@@ -340,6 +340,20 @@ li.active .icon-cozy-inline > svg {
...
+  justify-content: center;
+  align-items: center;
+  background-color: #fff;
+  margin-top: 0px;
+  padding-top: 15px;
+  padding-bottom: 15px;
+}
+
+.paper-illustration img {
+  height: 128px;
```
Comment (Ldoppea) [inline apps/browser/src/popup/scss/misc.scss:354]: I'm wondering where this value come from. It does not seem to appear in the figma template.

Also illustrations with landscape would be displayed too large, wouldn't they?

Should we instead set maxHeight and maxWidth?

**Review by Ldoppea — APPROVED** [review]
Nice one 👍

File: libs/cozy/fields.helper.ts (hunk 2 above)
Comment (zatteo) [reply libs/cozy/fields.helper.ts:67]: It's a prudence measure. If a label does not have a qualification type, or a known qualification type, I think it is better to do nothing.

File: apps/browser/src/popup/scss/misc.scss (hunk 3 above)
Comment (zatteo) [reply apps/browser/src/popup/scss/misc.scss]: Good question. I tried different thumbnails (with big width and small height and vice versa) and it was not displaying nicely.

I added a new commit to fix the issue.

**Pull request merged**